tools/pot/openvino/tools/pot/algorithms/quantization/accuracy_aware/README.md

```diff
@@ -1,12 +1,12 @@
 # AccuracyAwareQuantization Algorithm {#accuracy_aware_README}
 
 ## Introduction
-AccuracyAwareQuantization algorithm is aimed at accurate quantization and allows the model's accuracy to stay within the 
-pre-defined range. This may cause a 
-degradation in performance in comparison to [DefaultQuantization](../default/README.md) algorithm because some layers can be reverted back to the original precision.
+AccuracyAwareQuantization Algorithm purpouse is to accurate quantization and allowing the model's accuracy to stay within the 
+pre-defined range. In comparison to [DefaultQuantization](../default/README.md) algorithm this may cause a 
+degradation in performance because some layers can be reverted back to the original precision.
 
 ## Parameters
-Since the [DefaultQuantization](../default/README.md) algorithm is used as an initialization, all its parameters are also valid and can be specified. Here is an example of the definition of the `AccuracyAwareQuantization` method and its parameters:
+Since the [DefaultQuantization](../default/README.md) algorithm is used as an initialization, all its parameters are also valid and can be specified. Below is an example of lthe definition of the `AccuracyAwareQuantization` method and its parameters:
```

Comment:
```suggestion
Since the [DefaultQuantization](../default/README.md) algorithm is used as an initialization, all its parameters are also valid and can be specified. Below is an example definition of the `AccuracyAwareQuantization` method and its parameters:
```